modules/fedimint-wallet-common/src/lib.rs

```diff
@@ -309,6 +313,8 @@ pub enum WalletInputError {
     PegInAlreadyClaimed,
     #[error("The wallet input version is not supported by this federation")]
     UnknownInputVariant(#[from] UnknownWalletInputVariantError),
+    #[error("The pegin amount is not economically viable for the federation")]
+    PegInUnderMinimum,
```

Comment:
Can we add an error here with no default variant?

Reply:
Hmm, I believe we'd break compatibility with existing `0.3.*` and `0.4.0` clients even if we backport. It's not semantically correct, but perhaps we can reuse `UnknownInputVariant` to maintain compatibility.